src/js/components/FormField/FormField.js

```diff
@@ -356,6 +356,12 @@ const FormField = forwardRef(
           }
         : {};
 
+    let { requiredAsterisk } = theme.formField.label;
+    if (requiredAsterisk === true)
+      // a11yTitle necessary so screenreader announces as "required"
+      // as opposed to "star"
```

Comment:
Is there documentation to reference this?

Reply:
Done.